chalice/awsclient.py

```diff
@@ -90,22 +90,27 @@ def create_function(self,
             kwargs['Timeout'] = timeout
         if memory_size is not None:
             kwargs['MemorySize'] = memory_size
+        return self._call_client_method_with_retries(
+            self._client('lambda').create_function, kwargs)['FunctionArn']
+
+    def _call_client_method_with_retries(self, method, kwargs):
+        # type: (Callable[..., Any], Dict[str, Any]) -> Dict[str, Any]
```

Comment:
Can we tighten up the signature here?  Callable that only takes kwargs and returns a dict of `str, Any` maybe?  http://mypy.readthedocs.io/en/latest/kinds_of_types.html#extended-callable-types